ush/config.sh.RRFS_wcoss

```diff
@@ -1,123 +1,198 @@
 MACHINE="WCOSS_DELL_P3"
-ACCOUNT="HRRR-T2O"
```

Comment:
It's great that we have a configuration working on WCOSS. I'd prefer to see the name of this file be changed to be more descriptive of the configuration that it describes. It doesn't seem to be the same as any of the others. Maybe we should be adding a comment block to the top of each of the config files now that they have gotten very diverse.

Reply:
Yes. I agree. Comment block is add to the top of this configure files.
There are several configure files for different domains and machines. It should be able to be combined into an input namelist to control this in the future. Here, let's focus on spinup cycle with configure files used for WCOSS and work on the issue of code diverging later.

Reply:
This comment is not quite what I was getting at. I'd like to see more information about the configuration that is expected. Maybe a bit about the cycling strategy, forecast frequency and length, domain, etc. The goal is to help a user quickly understand what the default is on WCOSS since it is not similar to what is being run elsewhere.